src/main/resources/assets/bloodmagic/lang/fr_FR.lang

```diff
@@ -1,239 +1,250 @@
+item.BloodMagic.pack.selfSacrifice.name=Sac à dos Sanguin 
+item.BloodMagic.pack.sacrifice.name=Manteau des Bras
+item.BloodMagic.daggerOfSacrifice.name=Dague des Sacrifices
+
+item.BloodMagic.lavaCrystal.name=Cristal de Lave
+
+item.BloodMagic.bound.sword.name=Lame Sacrificielle
+item.BloodMagic.bound.pickaxe.name=Pioche Sacrificielle
+item.BloodMagic.bound.axe.name==Hache Sacrificielle
+item.BloodMagic.bound.shovel.name==Pelle Sacrificielle
+
+item.BloodMagic.bucket.lifeEssence.name=Seau de Vie
+
+item.BloodMagic.scribe.water.name=Craie d’Inscription Elémentaire: Eau
+item.BloodMagic.scribe.fire.name=Craie d’Inscription Elémentaire: Feu
+item.BloodMagic.scribe.earth.name=Craie d’Inscription Elémentaire: Terre
+item.BloodMagic.scribe.air.name=Craie d’Inscription Elémentaire: Air
+item.BloodMagic.scribe.dusk.name=Craie d’Inscription Elémentaire: Crépuscule
+item.BloodMagic.scribe.dawn.name=Craie d’Inscription Elémentaire: Aube
+
+item.BloodMagic.focus.weak.name=Focus de Téléposition
+item.BloodMagic.focus.enhanced.name=Focus de Téléposition Amélioré
+item.BloodMagic.focus.reinforced.name=Focus de Téléposition Renforcé
+item.BloodMagic.focus.demonic.name=Focus de Téléposition Diabolique
+
+item.BloodMagic.slate.blank.name=Tablette Vierge
+item.BloodMagic.slate.reinforced.name=Tablette Renforcée
+item.BloodMagic.slate.imbued.name=Tablette Imprégnée
+item.BloodMagic.slate.demonic.name=Tablette Diabolique
+item.BloodMagic.slate.ethereal.name=Tablette Éthérée
+
+item.BloodMagic.orb.weak.name=Orbe Sanguinaire Affaiblie
+item.BloodMagic.orb.apprentice.name=Orbe Sanguinaire de l’Apprenti
+item.BloodMagic.orb.magician.name=Orbe Sanguinaire du Magicien
+item.BloodMagic.orb.master.name=Orbe Sanguinaire du Maître
+item.BloodMagic.orb.archmage.name=Orbe Sanguinaire de l'Enchanteur
+item.BloodMagic.orb.transcendent.name=Orb Sanguinaire Transcendante
+
+item.BloodMagic.reagent.incendium.name=Incendium
+item.BloodMagic.reagent.magicales.name=Magicales
+item.BloodMagic.reagent.sanctus.name=Sanctus
+item.BloodMagic.reagent.aether.name=Aether
+item.BloodMagic.reagent.crepitous.name=Crepitous
+item.BloodMagic.reagent.crystallos.name=Crystallos
+item.BloodMagic.reagent.terrae.name=Terrae
+item.BloodMagic.reagent.aquasalus.name=Aquasalus
+item.BloodMagic.reagent.tennebrae.name=Tennebrae
+item.BloodMagic.reagent.offensa.name=Offensa
+item.BloodMagic.reagent.praesidium.name=Praesidium
+item.BloodMagic.reagent.orbisterrae.name=Orbis Terrae
+item.BloodMagic.reagent.virtus.name=Virtus
+item.BloodMagic.reagent.reductus.name=Reductus
+item.BloodMagic.reagent.potentia.name=Potentia
+
+item.BloodMagic.bloodShard.weak.name=Tesson Sanguin Faible
+item.BloodMagic.bloodShard.demon.name=Tesson Sanguin Diabolique
+
+item.BloodMagic.baseComponent.reagentWater.name=Réactif d'Eau
+item.BloodMagic.baseComponent.reagentLava.name=Réactif de Lave
+item.BloodMagic.baseComponent.reagentAir.name=Réactif d'Air
+item.BloodMagic.baseComponent.reagentFastMiner.name=Réactif de Miner
```

Comment:
This one sounds weird, maybe "Réactif Minier"? (I could also suggest "Réactif du Mineur", but that could be translated to both "Miner's reagent" or "Minor's reagent" ... )